community-operators/strimzi-kafka-operator/strimzi-cluster-operator.v0.11.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,548 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: strimzi-cluster-operator.v0.11.1
+  namespace: placeholder
+  annotations:
```

Comment:
```suggestion
  annotations:
    repository: https://github.com/strimzi/strimzi-kafka-operator
```

Reply:
Thanks. I added this also to the other CSV file for 0.11.1 to have them in sync.